packages/edit-widgets/src/components/more-menu/index.js

```diff
@@ -1,33 +1,20 @@
 /**
  * WordPress dependencies
  */
-import {
-	DropdownMenu,
-	MenuGroup,
-	MenuItem,
-	VisuallyHidden,
-} from '@wordpress/components';
+import { MenuGroup, MenuItem, VisuallyHidden } from '@wordpress/components';
 import { useState } from '@wordpress/element';
 import { __, _x } from '@wordpress/i18n';
-import { external, moreVertical } from '@wordpress/icons';
+import { external } from '@wordpress/icons';
+import { OptionsMenu, OptionsMenuFeatureToggle } from '@wordpress/interface';
 import { displayShortcut } from '@wordpress/keycodes';
 import { useShortcut } from '@wordpress/keyboard-shortcuts';
 import { useViewportMatch } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
-import FeatureToggle from './feature-toggle';
 import KeyboardShortcutHelpModal from '../keyboard-shortcut-help-modal';
 
-const POPOVER_PROPS = {
-	className: 'edit-widgets-more-menu__content',
-	position: 'bottom left',
-};
-const TOGGLE_PROPS = {
-	tooltipPosition: 'bottom',
-};
-
 export default function MoreMenu() {
```

Comment:
Should we also refactor `MoreMenu` in the edit post screen to use the new component?

https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/components/header/more-menu/index.js

Reply:
Yeah definitely, I plan to work through each editor after this to convert them.